src/components/Inputs.js

```diff
@@ -14,10 +14,17 @@ const Container = styled.div`
   margin-right: 10px;
 `;
 
-export const Dropdown = ({ name, options, selected, update }) => {
+export const Dropdown = ({
+  name,
+  options,
+  selected,
+  update,
+  className,
```

Comment:
What are you using className for? shouldn't really be needed with styled components and I don't see the param used anywhere in this pr

Reply:
I think that has to be done so that styled components can extend the component. ie. styled(Input) or something

Reply:
it's possible that the input isn't being extended in this way. It was a while ago now so I don't recall. I just know when I put a bunch of form elements in a row they never quite line up so I have to do a lot of adding some margin top/bottom to different things. So I'm guessing that's why the class name was there, but not sure if its necessary at the moment.